### ISSUE TITLE
prefer-describe-function-title false positive on type imports

### ISSUE THREAD

**Issue by leeyi45** [vitest-dev/eslint-plugin-vitest#954]
In Typescript it is possible to specify that something is only a "type" import - that is it gets elided at runtime, which means that the code below is an error:

```ts
import type { TestFunc } from '../types';

describe(TestFunc, () => {});
```
since `TestFunc` is just a type and can't be used as a value. But `prefer-describe-function-title` insists on converting the string `TestFunc` into an identifier:

```ts
import type { TestFunc } from '../types';

describe('TestFunc', () => {}); // is highlighted as an error
```
